python/ray/scripts/scripts.py

```diff
@@ -562,6 +563,16 @@ def create_or_update(cluster_config_file, min_workers, max_workers, no_restart,
     if restart_only or no_restart:
         assert restart_only != no_restart, "Cannot set both 'restart_only' " \
             "and 'no_restart' at the same time!"
+    if not os.path.isfile(cluster_config_file):
+        try:
+            response = urllib.request.urlopen(cluster_config_file, timeout = 5)
+            content = response.read()
+            f = open("downloaded_cluster_config.yaml", "wb")
+            f.write(content)
+            f.close()
```

Comment:
maybe use `with open`?

And then for the file name, can you use the file name from the original file?